use-timescale/continuous-aggregates/refresh-policies.md

```diff
@@ -31,15 +31,7 @@ Among others, `add_continuous_aggregate_policy` takes the following arguments:
     24 hours.
 
 If you set the `start_offset` or `end_offset` to `NULL`, the range is open-ended
-and extends to the beginning or end of time. However, it's recommended to set
-the `end_offset` so that at least the most recent time bucket is excluded. For
-time-series data that mostly contains writes that occur in time stamp order, the
-time buckets that see lots of writes quickly have out-of-date aggregates. You
-get better performance by excluding the time buckets that are getting a lot of
-writes.
-
-In addition, materializing the most recent bucket might interfere with
-[real-time aggregation][future-watermark].
+and extends to the beginning or end of time. If you set `end_offset` within the current time bucket, and [real-time aggregation][future-watermark] is disabled, the current time bucket is excluded. This is to improve performance: for time-series data that mostly contains writes that occur in the time stamp order, the time buckets that see lots of writes quickly have out-of-date aggregates. You get better performance by excluding the time buckets that are getting a lot of writes.
```

Comment:
FWIW: another reason is also that you cannot really refresh a partial bucket. You either compute the whole bucket or not at all.

Reply:
added, thank you